feature/policy_forwarding/otg_tests/mpls_gre_ipv4_encap_scale_test/mpls_gre_ipv4_encap_scale_test.go

```diff
@@ -0,0 +1,616 @@
+// Package mpls_gre_ipv4_encap_scale_test tests mplsogre encap functionality.
+package mpls_gre_ipv4_encap_scale_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/open-traffic-generator/snappi/gosnappi"
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/cfgplugins"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/featureprofiles/internal/iputil"
+	otgconfighelpers "github.com/openconfig/featureprofiles/internal/otg_helpers/otg_config_helpers"
+	otgvalidationhelpers "github.com/openconfig/featureprofiles/internal/otg_helpers/otg_validation_helpers"
+	"github.com/openconfig/featureprofiles/internal/otg_helpers/packetvalidationhelpers"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ondatra/netutil"
+	"github.com/openconfig/ygot/ygot"
+)
+
+// TestMain calls main function.
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+const (
+	ieee8023adLag                = oc.IETFInterfaces_InterfaceType_ieee8023adLag
+	greProtocol                  = 47
+	dutStartIP                   = "169.254.0.1"
+	otgStartIP                   = "169.254.0.2"
```

Comment:
nit - keep format the same.  You have similarly named vars for `IPv6` so these should have `IPv4`


```suggestion
	dutStartIPv4                   = "169.254.0.1"
	otgStartIPv4                   = "169.254.0.2"
```